estat/README.md

```diff
@@ -0,0 +1,156 @@
+## Overview
+
+`estat` - extensible performance observability tool for Delphix Dynamic Data Platform.  
+
+This tool mainly uses `eBPF` programs with `BCC` front-end to collect and output performance statistics from kernel via `kprobes`.  
```

Comment:
nit: 'BCC' -> 'a BCC'
nit: 'kernel' -> 'the kernel'

Also, here and in several other places, lines have trailing whitespace which seems to break the paragraph rendering.

Reply:
Those trailing blank spaces were intentional to cause a paragraph/break per markdown style.